composer.json

```diff
@@ -110,10 +109,10 @@
     },
     "require": {
         "cweagans/composer-patches": "^1.6.0",
-        "composer/installers": "^1.6",
+        "composer/installers": "^1.9",
         "oomphinc/composer-installers-extender": "^1.1",
-        "drupal-composer/drupal-scaffold": "^2.5.0",
```

Comment:
When using goalgorilla/social_template there is still a reference to ` "drupal-composer/drupal-scaffold": "^2.6.1"` in there which conflicts with this change. Without the aforementioned conflict composer will install / update nicely with the new scaffold.

Reply:
Yep, as mentioned in https://github.com/goalgorilla/open_social/pull/1889#issuecomment-649555596

The PR in https://github.com/goalgorilla/social_template/pull/24 will need to be merged with this PR to make sure that conflict doesn't happen.